podpac/core/cache/cache.py

```diff
@@ -7,6 +7,7 @@
 import os
 from glob import glob
 import shutil
+from hashlib import md5 as hash_alg
```

Comment:
@drc-creare @dsully1992 Note that Python 3's default `hash` function uses a random seed. That probably why you didn't get consistent hashes on the Lambda functions, David. Dave, this change was required to actually make the disk cache useable across Python sessions. A little tricky to test in a unittest, so I didn't do it.